packages/salesforcedx-sobjects-faux-generator/test/integration/mockData.ts

```diff
@@ -108,11 +108,11 @@ export const mockAPIResponse = {
         undeletable: false,
         updateable: false,
         urls: {
-          rowTemplate: '/services/data/v46.0/sobjects/ApexPageInfo/{ID}',
+          rowTemplate: '/services/data/v50.0/sobjects/ApexPageInfo/{ID}',
           defaultValues:
-            '/services/data/v46.0/sobjects/ApexPageInfo/defaultValues?recordTypeId&fields',
-          describe: '/services/data/v46.0/sobjects/ApexPageInfo/describe',
-          sobject: '/services/data/v46.0/sobjects/ApexPageInfo'
+            '/services/data/v50.0/sobjects/ApexPageInfo/defaultValues?recordTypeId&fields',
+          describe: '/services/data/v50.0/sobjects/ApexPageInfo/describe',
+          sobject: '/services/data/v50.0/sobjects/ApexPageInfo'
```

Comment:
In `packages/system-tests/assets/sfdx-simple/functions/demoJavaScriptFunction/test/unitTests.js`, there are a few references to `/services/data/v48.0/sobjects/...`

Should they be updated to `v50.0` as well?  It's just mock data in an integration test, so it probably doesn't matter, but I wanted to raise the issue just in case.

Reply:
I can update them